pkg/declarativeconfig/transform/auth_provider_test.go

```diff
@@ -365,6 +369,8 @@ func TestUniversalTransformAuthProvider(t *testing.T) {
 		oidc.ClientIDConfigKey:                  authProvider.OIDCConfig.ClientID,
 		oidc.ClientSecretConfigKey:              authProvider.OIDCConfig.ClientSecret,
 		oidc.DisableOfflineAccessScopeConfigKey: "true",
+		oidc.DontUseClientSecretConfigKey:       "false",
+		oidc.ExtraScopesConfigKey:               "",
```

Comment:
Would it make sense to add a test with non-empty extra scopes? What's the format? Comma separated list of words?